lambdatimes/src/components/Content/Card.js

```diff
@@ -1,19 +1,26 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
 const Card = props => {
   return (
     <div className="card">
-      <div className="headline">{/* headline goes here */}</div>
+      <div className="headline">{props.card.headline}</div>
       <div className="author">
         <div className="img-container">
-          <img src={'' /* image source goes here */} />
+          <img src={props.card.img} />
```

Comment:
Always add an `alt=""` when using an `img` tag. It's more semantic and removes a console warning.